.github/CONTRIBUTING.md

```diff
@@ -37,13 +37,14 @@ This means you care about the quality of your contributions.
     * When adding a new aport: testing/apkname: new aport
     * When modifying an aport: testing/apkname: short description about changes
   * If needed provide a proper formatted (line wrapped) description of what your patch will do. You can provide a description in the PR, but you must include a message for this specific commit in the commit description. If in the future we would like to distance ourselves from Github the PR information could be lost.
-5. Open your copy of the aports repository at github.com and switch to your feature branch. You should now see an option to create your PR. [More info](https://help.github.com/articles/creating-a-pull-request/)
-6. Wait for an Alpine Linux developer to review your changes.
-7. If all is ok your PR will be merged but if a developer asks for changes please do as follows:
+5. Add your branch to your copy of the aports repository on GitHub `git push -u origin my_new_feature`.
+6. Open your copy of the aports repository at github.com and switch to your feature branch. You should now see an option to create your PR. [More info](https://help.github.com/articles/creating-a-pull-request/)
+7. Wait for an Alpine Linux developer to review your changes.
+8. If all is ok your PR will be merged but if a developer asks for changes please do as follows:
   * Make the requested changes.
   * Add your file(s) to git and commit (we will squash your commits if needed).
   * Push your changes `git push origin my_new_feature`.
```

Comment:
shouldnt this also be `git push -u origin my_new_feature`, with `-u`?

Reply:
My reading of `-u` is that it's only needed for initial setup:
```
       -u, --set-upstream
           For every branch that is up to date or successfully pushed, add
           upstream (tracking) reference, used by argument-less git-pull(1)
           and other commands. For more information, see branch.<name>.merge
           in git-config(1).
```
So if you've already done it once, then you're already set up on both ends.

Reply:
Correct, `-u` is only necessary when pushing a branch for the first time

Reply:
Well, unless you have `current` as `push.default` configured for git

Reply:
Even when you don't have `push.default` set to `current`, it's not required, it's just convenient to have so that git can give you more information :wink:.